packages/carequality-sdk/src/client/carequality-fhir.ts

```diff
@@ -203,13 +190,12 @@ export class CarequalityManagementAPIImpl implements CarequalityManagementAPI {
    * @param oid string containing the organization OID
    * @returns an XML string containing an OperationOutcome resource - see Carequality documentation for details - https://carequality.org/healthcare-directory/OperationOutcome-create-success-example2.xml.html
    */
-  async updateOrganization(org: string, oid: string): Promise<string> {
+  async updateOrganization({ org, oid }: UpdateOrganizationParams): Promise<Organization> {
     const query = new URLSearchParams();
     query.append("apikey", this.apiKey);
-    query.append("_format", XML_FORMAT);
 
-    const url = `${CarequalityManagementAPIImpl.ORG_ENDPOINT}/${oid}?${query.toString()}`;
-    const resp = await this.sendPutRequest(url, org, { "Content-Type": "application/xml" });
+    const url = `${CarequalityManagementAPIImplFhir.ORG_ENDPOINT}/${oid}?${query.toString()}`;
+    const resp = await this.sendPutRequest(url, org, { "Content-Type": "application/json" });
```

Comment:
since all three functions are now using the same content type:
`"Content-Type": "application/json"`, 

let's just set in on the `buildHeaders` function